src/com/company/Assig4.java

```diff
@@ -271,6 +271,18 @@ private void cleanImage() {
       }
       return shiftedArray;
    }
+   
+   /**
+    * Utility method that sets the image to white (false).
+    */
+   private void clearImage() {
```

Comment:
Do you mind changing this method to iterate through `this.image` and set the values to false using `image.setPixel`?

We should be able to iterate through `image` similar to [this way](https://github.com/erickakoyama/cst338-assig4/blob/dd0dc3992ebbc2fa6449af148f7b5da03556d623/src/com/company/Assig4.java#L332).

Reply:
Ah- yep! Sorry, misunderstood the instructions